Css transition for expandable block (#1396)
iTwin/iTwinUI: .changeset/itchy-poems-shout.md

> ---
> '@itwin/itwinui-react': minor
> '@itwin/itwinui-css': minor
> ---
> 
> Expandable block animation now uses pure css for transitioning and `WithCSSTransition` has been removed.

Reviewer: should be a patch change and should not mention internal names like "WithCSSTransition"

```suggestion
---
'@itwin/itwinui-react': patch
'@itwin/itwinui-css': patch
---

Expandable block animation now uses pure css instead of javascript for transitioning.
```
